TRB_label_assist/clustering.py

```diff
@@ -28,9 +29,26 @@
     'mean_shift'
 ]
 
+def CleanEntryTypeData(loc_df,loc_entry):
+
+    """
+    Helps weed out entries from the list of entries who's which were removed from the df using
+    esdtq.filter_labeled_trips() and esdtq.expand_userinputs()
+
+    loc_df : dataframe amde from entry type data
+    loc_entry : the entry type equivalent of loc_df ,
+                which was passed alongside the dataframe while loading the data
+
+    """
+
+    ids_in_df=set(loc_df['_id'])
+    filtered_loc_entry= [entry for entry in loc_entry if entry['_id'] in ids_in_df ]
```

Comment:
have you run this code? Don't you need an else for the if?
You might want to try `filter` instead